mmv1/products/cloudids/terraform.yaml

```diff
@@ -0,0 +1,31 @@
+# Copyright 2022 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Provider::Terraform::Config
+overrides: !ruby/object:Overrides::ResourceOverrides
+  Endpoint:  !ruby/object:Overrides::Terraform::ResourceOverride
+    id_format: 'projects/{{project}}/locations/{{location}}/endpoints/{{name}}'
+    base_url: projects/{{project}}/locations/{{location}}/endpoints
+    import_format: ["projects/{{project}}/locations/{{location}}/endpoints/{{name}}"]
+    autogen_async: true
+    examples:
+      - !ruby/object:Provider::Terraform::Examples
+        name: "cloudids_endpoint"
+        primary_resource_id: "example-endpoint"
+        skip_test: true
```

Comment:
The test itself requires creating a network resource, and it takes about 25-30 minutes to run the test. The example here is nearly identical the test I added, that's why I added skip_test.

Reply:
Cool. If we just had the two identical tests, I would vouch for deleting the handwritten one in favor of the generated (since the documentation is linked to the generated). But since we can add update steps to the handwritten one, I'm fine with leaving this skipped. mind adding a quick comment to the `skip_test` line explaining the reasoning just for information's sake?

Reply:
Comment added.